daemon/api_systems.go

```diff
@@ -44,7 +45,10 @@ var systemsCmd = &Command{
 }
 
 var systemsActionCmd = &Command{
-	Path:        "/v2/systems/{label}",
+	Path:       "/v2/systems/{label}",
+	GET:        getSystemDetails,
```

Comment:
I totally agree `getSystemAction` was a bit strange, but now there is a mismatch between get/post naming which I really don't like. Could we get `postSystemsAction` renamed in a followup? Also it's a bit nitpicky, but I don't like there isn't a theme with `getAllSystems`. 

Wouldn't it make more sense to keep a common theme in them, like `getAllSystemDetails` and `getSystemDetails`, or maybe something like `getSystems` and `getSystemByLabel` would be even more descriptive as to their meaning.

Reply:
Thanks, that is good input. Happy to do this as followup as suggested.

Reply:
to be clear getSystemDetails really provides more details that a single entry in the response of getAllSystems so making those names completely match is not truthful either

Reply:
Ah ok, then maybe ignore that part, thanks for explaining